PLSS/profiles/build.profile.js

```diff
@@ -20,7 +20,9 @@ var profile = {
                 'dojox/gfx/svg',
                 'dojox/gfx/svgext',
                 'dojox/gfx/shape',
-                'esri/dijit/Attribution'
+                'esri/dijit/Attribution',
+                'esri/layers/VectorTileLayerImpl',
+                'esri/layers/vectorTiles/views/vectorTiles/WorkerTileHandler'
```

Comment:
**question**: didn't this break the build?

Reply:
No, `esri/layers/vectorTiles/core/workers/worker` was the one causing issues.